src/vs/base/parts/quickinput/browser/quickInput.ts

```diff
@@ -1249,6 +1260,7 @@ export class QuickInputController extends Disposable {
 
 		const inputBox = this._register(new QuickInputBox(filterContainer));
 		inputBox.setAttribute('aria-describedby', `${this.idPrefix}message`);
+		inputBox.setAttribute('aria-live', 'polite');
```

Comment:
I tested this change out and used VoiceOver on macOS and it didn't seem to have any affect... Even without this change, the error message was read out loud.

Reply:
I observed (with ChromeVox on Linux) that when the initial value is OK but the new value is invalid (causing an error message to appear), the error message is read in real time only when the aria-live is true. Hence this change.

Reply:
That's fair. I'll take your word.